CONTRIBUTING.md

```diff
@@ -182,6 +182,13 @@ $ gcloud kms keys create key2 --keyring $MY_KEYRING --location $MY_LOCATION --pu
 $ export GCLOUD_TESTS_GOLANG_KEYRING=projects/$GCLOUD_TESTS_GOLANG_PROJECT_ID/locations/$MY_LOCATION/keyRings/$MY_KEYRING
 # Authorizes Google Cloud Storage to encrypt and decrypt using key1.
 $ gsutil kms authorize -p $GCLOUD_TESTS_GOLANG_PROJECT_ID -k $GCLOUD_TESTS_GOLANG_KEYRING/cryptoKeys/key1
+# Authorizes Google Cloud Bigtable to encrypt and decrypt using key1
+gcloud kms keys add-iam-policy-binding key1 \
+    --keyring $MY_KEYRING \
+    --location $MY_LOCATION \
+    --role roles/cloudkms.cryptoKeyEncrypterDecrypter \
+    --member allAuthenticatedUsers \
```

Comment:
is this true? Wouldn't you want this to point to a service account?

Reply:
I figured that, as this encryption key isn't really a secret, opening it up to all authn'd users was fine. 

@tritone do you know what `gsutil kms` is doing behind the scenes?

Reply:
Hmm not exactly sure what the question is here but here's the code: https://github.com/GoogleCloudPlatform/gsutil/blob/7845859b924004d16db0408335fd434424f6c8d7/gslib/commands/kms.py

Looks like it requests a service account and creates one if it doesn't exist.

Reply:
Could you re-use the service account used elsewhere in the setup? `GCLOUD_TESTS_GOLANG_KEY` should already point to a service account json; is there a gcloud command to grab the email from that? I see https://cloud.google.com/sdk/gcloud/reference/iam/service-accounts/list . I just don't know enough about IAM to be confident that adding this binding isn't some kind of security risk.

Reply:
So, the complication here is we store the path to the key, but the email you need (the member id) for the service account is within the json.

```
  "client_email": "{PROJECT}@{PROJECT}.iam.gserviceaccount.com",
```

I could infer this as it generally follows that form, but it may not.  I was avoiding adding another EnvVar.